www/docs/providers/credentials.md

```diff
@@ -53,7 +55,7 @@ providers: [
       username: { label: "Username", type: "text", placeholder: "jsmith" },
       password: {  label: "Password", type: "password" }
     },
-    async authorize(credentials) {
+    async authorize(credentials, req) {
```

Comment:
Found some additional places where the docs should be updated:
- https://next-auth-git-fork-igordanchenko-feat-add-request-t-297b00.vercel.app/providers/credentials#example-code
- https://next-auth-git-fork-igordanchenko-feat-add-request-t-297b00.vercel.app/tutorials/ldap-auth-example

Reply:
Updated these docs in a separate commit